pytest.ini

```diff
@@ -1,5 +0,0 @@
-[pylama]
-format = pylint
-skip = */doc/*,*/data/*
-linters = pep8,pyflakes
-ignore = E501,E251,E221,E201,E202,E203
```

Comment:
Need to migrate these to the pre-commit hooks. I'll  that in another PR and fix the pre-commits failures at the same time.